sections/docker/image-tags.md

```diff
@@ -0,0 +1,29 @@
+# Understand image tags vs digests, and use the "latest" tag with caution
+
+### One Paragraph Explainer
+
+If this is a production situation, and security and stability are important, then just "convenience" is likely not the best deciding factor.
+
+In addition, the 'latest' tag is Docker's default tag. This means that a developer who forgets to add an explicit tag, will accidentally push a new version of an image as 'latest', which might end in very unintended results if the latest tag is being relied upon as the latest production image.
+
+### Code example:
+
+```bash
+$ docker build -t company/image_name:0.1 .
+# :latest image is not updated
+$ docker build -t company/image_name
+# :latest image is updated
+$ docker build -t company/image_name:0.2 .
+# :latest image is not updated
+$ docker build -t company/image_name:latest .
+# :latest image is updated
+```
+
+### What Other Bloggers Say
+From the blog by [Vladislav Supalov](https://vsupalov.com/docker-latest-tag/):
+> Some people expect that :latest always points to the most-recently-pushed version of an image. That’s not true.
+
+From the [Docker success center](https://success.docker.com/article/images-tagging-vs-digests)
+> 
```

Comment:
Good link! Hard to chose what to quote
Maybe the whole block ?

> ### Tagging is best practice because
>
>    * Human-readable versioning of your images
>    * Easier to run a specific version of an image
>    * Can aid in the process (a release is tagged once it is verified/approved for production)
>
> ### Pinning-by-digest is best practice because
>
>    * Tags are mutable, so there is no guarantee a tag will never change.
>    * It guarantees that every instance of the service is running exactly the same code.
>    * It allows you to put an image through QA/testing and verify that that version of the image is approved to go into production.
>    * You can use Docker Content Trust and sign specific versions of the image.
>    * You can roll back to an earlier version of the image, even if that version was not tagged (or "no longer tagged").
>    * Digests also prevent race-conditions; if a new image is pushed while a deploy is in progress, different nodes may be pulling the images at different times, so some nodes have the new image, some have the old one.
>
> Manually specifying a digest is a bit involved. That's why services automatically resolve tags to digests to make this easy to use.